roles/openshift_logging_elasticsearch/tasks/full_cluster_restart.yml

```diff
@@ -0,0 +1,162 @@
+---
+# Disable external communication for {{ _cluster_component }}
+- name: Disable external communication for logging-{{ _cluster_component }}
+  oc_service:
+    state: present
+    name: "logging-{{ _cluster_component }}"
+    namespace: "{{ openshift_logging_elasticsearch_namespace }}"
+    selector:
+      component: "{{ _cluster_component }}"
+      provider: openshift
+      connection: blocked
+    labels:
+      logging-infra: 'support'
+    ports:
+    - port: 9200
+      targetPort: "restapi"
+
+# Flush ES
+# This is documented as a best effort, if it fails, we are okay with that
```

Comment:
How are we able to do this if we just disabled the service in the previous task?

Reply:
good catch... we'll have to make this make calls from the pod